CMakeLists.txt

```diff
@@ -14,10 +14,7 @@ option(nanopb_BUILD_RUNTIME "Build the headers and libraries needed at runtime"
 option(nanopb_BUILD_GENERATOR "Build the protoc plugin for code generation" ON)
 option(nanopb_MSVC_STATIC_RUNTIME "Link static runtime libraries" ON)
 
-find_program(nanopb_PROTOC_PATH protoc)
-if(NOT EXISTS nanopb_PROTOC_PATH)
-    message(FATAL_ERROR "protoc compiler not found")
-endif()
+find_program(nanopb_PROTOC_PATH protoc REQUIRED)
```

Comment:
I see there is a protoc binary bundled with nanopb. Should I add a path to that binary in case protoc is not found on the users machine?

Reply:
Hmm yeah, it would be good to have `generator-bin` first in the search path if it exists. Presumably those who download a binary package want to use the generator that comes with it.

Reply:
Do you want me to add a hint to CMake to search in `nanopb/generator/` if protoc was not found in the default paths on the user machine?

Reply:
Yeah, that could be last in the path. The `protoc` script in `generator` folder is just a wrapper around `grpcio-tools` Python package.